deploy/helm/metacontroller/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: v2.0.10
```

Comment:
do not need to update this, the release automation will bump it, please restore it as it was